mmv1/products/apigee/api.yaml

```diff
@@ -175,9 +175,14 @@ objects:
         description: |
           The size of the CIDR block range that will be reserved by the instance.
```

Comment:
Can we change this field from an Enum to a string and put a link to the documentation where the values are defined in the description. This will help us to prevent having to manage the maintenance regularly.

Reply:
Done